src/test/resources/Features/UserProfile.feature

```diff
@@ -0,0 +1,12 @@
+
+@UserProfile
+Feature: Title of your feature
+  I want to use this template for my feature file
+
+Scenario: To verify user details are saved and can upload Profile photo
+    Given Login with an user and navigate to My Account Settings page
+  	Then Click on Edit icon and update user details
```

Comment:
Use standard gherkin statements
Ex Given, When, And, Then

Reply:
Changes done